pkgs/development/python-modules/gcsfs/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, fetchFromGitHub, lib, pytestCheckHook, google-auth
+, google-auth-oauthlib, requests, decorator, fsspec, ujson, aiohttp, crcmod
+, pytest-vcr, vcrpy }:
+
+buildPythonPackage rec {
+  pname = "gcsfs";
+  version = "2021.04.0";
+
+  # github sources needed for test data
+  src = fetchFromGitHub {
+    owner = "dask";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-qwLi4+rNSAneQbbq2ljf58jDluVo6b/2SUwUNpFb1/Y=";
```

Comment:
Hash is wrong.

Reply:
Weird. I wonder if upstream moved the tag to another commit. I've updated it.

Reply:
You should ask them. If I did, that would be reason enough for me not to allow this package into nixpkgs.

Reply:
IMO this package is professionally maintained. The hashing error was my mistake. Github shows the tag was last updated before I wrote my original commit.